src/main/java/org/logstash/beats/BeatsHandler.java

```diff
@@ -54,8 +56,14 @@ public void channelRead0(ChannelHandlerContext ctx, Batch batch) throws Exceptio
 
     @Override
     public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) {
+        InetSocketAddress remoteAddress = (InetSocketAddress) ctx.channel().remoteAddress();
+
+        if (remoteAddress != null) {
+            logger.error("Exception: " + cause.getMessage() + ", from: " + remoteAddress.toString());
```

Comment:
I think you are right here, It would be OK to use `.info`, error level might just create more unnecessary issues than we actually require.